web-frontend/src/i18n/en_US.json

```diff
@@ -93,5 +97,8 @@
   "receive": {
     "account-receive": "Generate a link",
     "account-address": "Account address: "
+  },
+  "send-redirect": {
+    "redirecting": "Hey XX, we're transfering you to the send coin page"
```

Comment:
Maybe you need to split in two here.

```suggestion
    "redirecting-pre": "Hey "
    "redirecting-pos": ", we're transferring you to the send coin page"
```

Then you can interpolate in your code

Reply:
...or use the official i18n lib ^^

Reply:
We can use the support for interpolation now here!

Reply:
yep it's done

Reply:
Thank you @Pierregvx ! 🙏